bin/minifollowups/pycbc_sngl_minifollowup

```diff
@@ -171,30 +186,33 @@ if len(trigs.snr) == 0:
                   transformation_catalog_path=args.transformation_catalog)
     sys.exit(0)
 
+logging.info('finding loudest clustered events')
 trigs.mask_to_n_loudest_clustered_events(n_loudest=num_events,
                                       ranking_statistic=args.ranking_statistic)
-
 if len(trigs.stat) < num_events:
     num_events = len(trigs.stat)
 
 times = trigs.end_time
 tids = trigs.template_id
 
 # loop over number of loudest events to be followed up
-for num_event in range(num_events):
+order = trigs.stat.argsort()
```

Comment:
The triggers were previously ordered. Has this changed?

Reply:
It is not ordered, and can't be if we want to be able to read from the file quickly.